eventcore/src/executor/typestate.rs

```diff
@@ -663,6 +672,11 @@ impl<C: Command, ES: EventStore> ExecutionScopeWithWrites<C, ES> {
             .cloned()
             .collect()
     }
+    
+    /// Check if additional streams were requested (alias for functional pipeline)
+    pub fn check_additional_streams(&self, stream_resolver: &StreamResolver) -> Vec<StreamId> {
+        self.needs_additional_streams(stream_resolver)
+    }
```

Comment:
Not sure what the point of this function is? Isn't it just an alias for the needs_additional_streams function?